### ISSUE TITLE
Generate project add option: template-engine

### ISSUE THREAD

**Issue by dreamsxin** [phalcon/phalcon-devtools#66]
For example:

```
Options:
 --name                 Name of the new project
 --enable-webtools      Determines if webtools should be enabled [optional]
 --directory=s          Base path on which project will be created [optional]
 --type=s               Type of the application to be generated (micro, simple, multi-module)
 --template-path        Specify a template path [optional]
 --template-engine=s    Define the template engine, default php (php, volt)
 --use-config-ini       Use a ini file as configuration file [optional]
 --trace                Shows the trace of the framework in case of exception. [optional]
 --help                 Shows this help
```

**Comment by sergeysviridenko** [comment]
I suppose it's for creating new project?

**Comment by sergeyklay** [comment]
Implemented in the 3.2.x branch.
